src/Microsoft.AspNetCore.Mvc.TagHelpers/FormActionTagHelper.cs

```diff
@@ -194,7 +208,8 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
                 if (Route == null)
                 {
                     var urlHelper = UrlHelperFactory.GetUrlHelper(ViewContext);
-                    var url = urlHelper.Action(Action, Controller, routeValues);
+                    //add a null check for fragment?
```

Comment:
The convention is to add a `[Design]` prefix to a PR title if it's something you'd want people to vet without digging in to it